kahuna/public/js/components/gr-panel/gr-panel.html

```diff
@@ -31,7 +31,7 @@ <h1 class="panel__title">{{ctrl.selectedImages.size}} image<span ng:if="ctrl.sel
         </ul>
 
         <div class="image-info__group">
-            <div class="image-info__wrap metadata-line drop-menu image-info__usage-rights"
+            <div class="image-info__wrap metadata-line drop-menu image-info__usage-rights metadata__key"
```

Comment:
`metadata__key` here is a bit odd because according to [BEM](https://css-tricks.com/bem-101/) it means "the `key` child element of the `metadata` block", but there is no parent `metadata` block here.